### ISSUE TITLE
[ci] Workflows still refer to Java 8

### ISSUE THREAD

**Issue by kriegfrj** [bndtools/bnd#5873]
https://github.com/bndtools/bnd/blob/656a3d22636842fc92f621a5971730010c60737f/.github/workflows/cibuild.yml#L72-L78

https://github.com/bndtools/bnd/blob/656a3d22636842fc92f621a5971730010c60737f/.github/workflows/rebuild.yml#L103-L109

I'm pretty sure that this needs to be fixed, wasn't sure which of two options we should use:
1. Simply drop the "8 + " part, or,
2. change 8 to 17?

Thoughts? Do we need the baseline version too? I think @bjhargrave was responsible for this setup so if he could weigh in briefly that would help.

**Comment by pkriens** [comment]
Can't you just use the files from master? Or am I missing something?

**Comment by kriegfrj** [comment]
> Can't you just use the files from master? Or am I missing something?

The build files I linked to are on Master and they still reference Java 8.

Perhaps it has something to do with the p2 code?

**Comment by pkriens** [comment]
No, master looks different but does include 8.

Yes, I recall we need an 8 VM for the p2 generation.

**Comment by bjhargrave** [comment]
> Yes, I recall we need an 8 VM for the p2 generation.

Correct. Until the we fix the p2 update site generation in org.bndtools.p2, we still need a java 8 vm to build the p2 update site.

https://github.com/bndtools/bnd/blob/7e2ac5d012aae94a3d2e3d99d5d8049579acba0c/org.bndtools.p2/build.gradle#L99-L101
